ftw/showroom/js/src/register.js

```diff
@@ -27,12 +27,16 @@ export default function Register(items = [], options) {
   function checkPointer() {
     oberserver.update(pointer);
     if(oberserver.hasChanged()) {
-      if (pointer === 0) {
-        options.head(reveal.current);
-      }
-      if (pointer === reveal.size - 1) {
-        options.tail(reveal.current);
-      }
+      performCalls();
+    }
+  }
+
+  function performCalls() {
+    if (pointer === 0) {
+      options.head(reveal.current);
```

Comment:
but this is a property which cannot be called (this is the register, not the showroom which also has a current method 😉 )